lib/y2logsstep.pm

```diff
@@ -5,7 +5,8 @@ use strict;
 
 sub use_wicked() {
     script_run "cd /proc/sys/net/ipv4/conf";
-    script_run "for i in *[0-9]; do echo BOOTPROTO=dhcp > /etc/sysconfig/network/ifcfg-\$i; wicked --debug all ifup \$i; done";
+    script_run("for i in *[0-9]; do echo BOOTPROTO=dhcp > /etc/sysconfig/network/ifcfg-\$i; wicked --debug all ifup \$i; done", 300);
```

Comment:
Yes, we do not care whether it pass or fail. And actually sometimes it just fails with return code of 162.